core/models/contract.py

```diff
@@ -136,7 +136,7 @@ def to_dict(self):
         base_dict = {
             'id': self.id,
             'comments': self.comments,
-            'project': self.project,
+            'project': self.project.id,
```

Comment:
are we sure that we won't break any exports by this change?

Reply:
this should be ok - contract was not used in exports yet.